### DISCUSSION TITLE
Transfer of ownership to Amber Framework organization

### DISCUSSION THREAD

**Discussion by ArFnds** [amberframework/liquid.cr#63] [Announcements]
Hello,

As discussed #61 , I will transfer the ownership of this repository to the @amberframework organization.

I plan to do this on Saturday ( 19/11/2022 ).

Thanks all for still maintaining this project.
👋

**Comment by ArFnds** [comment]
Well i would like to :
![image](https://user-images.githubusercontent.com/112687932/202888795-8d8f0d30-f6e3-4375-91d7-00bc4a25196b.png)

Someone can unlock this ?

**Reply by crimson-knight** [reply]
@ArFnds I've gone ahead and sent you an invite to be a team member on the Amber org, hopefully that allows you to transfer into Amber. If not, you may need to add me to the Liquid.cr and then I can transfer directly into the AmberFramework org

**Reply by ArFnds** [reply]
It did not work. I gave you the rights

**Reply by crimson-knight** [reply]
All set, thank you very much!

**Comment by hugopl** [comment]
Now the code is under Amber umbrella, I would like to step in to be the maintaner once I get 80% of [golden liquid](https://github.com/jg-rp/golden-liquid) tests passing.

Currently only 58% of the tests are passing with my PR #69 that is becoming  almost a rewrite of the project... the PR introduces a lot of breaking changes... but my target is to be 100% compatible with ruby shopify liquid, not with original liquid.cr.

**Reply by crimson-knight** [reply]
@hugopl I've increased your permission levels to be a maintainer. You're doing a great job, and with the recent PR merge with the work you've done I can step in and help with adding support for the missing filters.

Once we reach 100% parity with Ruby liquid, I'd like to make Liquid the default view rendering for Amber and see if we can drive more usage of the shard.

**Reply by hugopl** [reply]
Nice, thanks.

About having liquid as default view rendering I think it's not a good idea, since the view code will be like ECR files, with a lot of HTML noise... IMO slang works better... or maybe something that is used in other language web frameworks like Slim, Haml, etc...

A year ago I started a HAML engine in Crystal... but I have too many pet projects to feed so I stopped it at some time.
